### DISCUSSION TITLE
Is there a simple way to split a 3D object using a 2D object?

### DISCUSSION THREAD

**Discussion by bikemike** [gumyr/build123d#686] [Q&A]
I'd like to split an object in a way similar to the "Slice Apart" feature in FreeCAD. In algebra mode I was hoping to use subtract to split a Part by a Sketch but I get an error:

"Only shapes with equal or greater dimension can be subtracted: not Part (3D) and Sketch (2D)"

If I call thicken on the Sketch I am able to split the Part but then the volumes of the split Parts does not equal the volume of the whole.

**Comment by gumyr** [comment]
One can [split](https://build123d.readthedocs.io/en/latest/operations.html#operations_generic.split) by any `Plane`.  I'm not aware of what "Slice Apart" does, could you provide a brief description of what you're trying to do?

**Reply by bikemike** [reply]
It allows you to split shapes by intersection with other shapes. I would like to be able to cut large object into smaller parts to be printed but I don't just want flat cuts. I would like it to be a dovetail or something similar for alignment and strength.

The image below from the FreeCAD wiki shows splitting an object using two faces.

![900px-Part_Slice_Demo](https://github.com/user-attachments/assets/49e3633c-57e2-42af-8f6a-bef927d736ef)

https://wiki.freecad.org/Part_SliceApart

**Comment by gumyr** [comment]
Got it. There is a [PR](https://github.com/gumyr/build123d/pull/427) that started this feature but it isn't complete - so no this doesn't exist currently.  I believe the Prusa Slicer supports doing this if that's any help.
